yarn.config.cjs

```diff
@@ -0,0 +1,121 @@
+/**
+ * The constraints defined in https://github.com/babel/babel/blob/main/yarn.config.cjs were used as a starting
+ * point for this file, see also: https://yarnpkg.com/features/constraints
+  */
+
+/** @type {import('@yarnpkg/types')} */
+const { defineConfig } = require('@yarnpkg/types');
+
+/**
+ *  Enforces consistent package metadata
+  */
+function enforcePackageInfo({ Yarn }) {
+  for (const workspace of Yarn.workspaces()) {
+    const info = {
+      author: "Embrace <support@embrace.io> (https://embrace.io/)",
+      bugs: {
+        "url": "https://github.com/embrace-io/embrace-react-native-sdk/issues",
+        "email": "support@embrace.io",
+      },
+      homepage: "https://github.com/embrace-io/embrace-react-native-sdk",
+      license: "Apache-2.0",
+      repository: {
+        type: "git",
+        url: "https://github.com/embrace-io/embrace-react-native-sdk",
+        directory: workspace.cwd,
+      },
+      publishConfig: {
+        access: "public",
+      },
+    };
+
+    for (const key of Object.keys(info)) {
+      if (workspace.manifest.private) {
+        workspace.unset(key);
+      } else {
+        workspace.set(key, info[key]);
+      }
+    }
+
+    if (workspace.manifest.private) {
+      workspace.unset("keywords")
+    } else {
+      workspace.set("keywords", [...new Set([
+        ...(workspace.keywords || []),
+        "embrace",
+        "react-native",
+        "tracking",
+        "observability",
+        "instrumentation",
+        "telemetry",
+        "bug tracker",
```

Comment:
Now that it is a yarn workspace you can se this, dependencies and peerDependency, hash and other things from here instead of having this in each package json.
You can delete this + peer dependencies + other common things from the inidivual json packages

Reply:
this works as a linter/validation steps so the actual values do end up getting written to the individual `package.json` files but that should happen automatically we'll just have to worry about keeping this file up to date